pkg/cmd/tui.go

```diff
@@ -0,0 +1,42 @@
+/*
+Copyright © 2020 Doppler <support@doppler.com>
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+	http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+package cmd
+
+import (
+	"github.com/DopplerHQ/cli/pkg/configuration"
+	tuiApp "github.com/DopplerHQ/cli/pkg/tui"
+	"github.com/DopplerHQ/cli/pkg/utils"
+	"github.com/spf13/cobra"
+)
+
+var tuiCmd = &cobra.Command{
+	Use:   "tui",
+	Short: "Launch TUI",
+	Args:  cobra.NoArgs,
+	Run:   tui,
+}
```

Comment:
Is there some way we can make it apparent that this is in alpha and this could disappear at any time?

Reply:
I changed the text to `Launch TUI (alpha - may change or disappear in the future)`

Reply:
Was thinking about this a bit more. Is there a phase in which we'll transition this from alpha to beta before GA? If not, I think we can label this command `Launch TUI (BETA)`.